examples/python/routing_example.py

```diff
@@ -72,21 +72,9 @@ def draw_graph(coupling_map):
 alternative_cube_architecture = SquareGrid(2, 2, 2)
 draw_graph(alternative_cube_architecture.coupling)
 
-# Other Architecture generators are supported including the HexagonalGrid:
-
-from pytket.routing import HexagonalGrid
-
-hex_architecture = HexagonalGrid(1, 2, 1)
-print(hex_architecture)
-draw_graph(hex_architecture.coupling)
-
-# and the CyclicButterfly:
-
-from pytket.routing import CyclicButterfly
-
-cb_architecture = CyclicButterfly(3)
-print(cb_architecture)
-draw_graph(cb_architecture.coupling)
+# In many cases, we are interested in the architectures of real devices.
+# These are available directly from the device backends, available within tket's respective extension packages.
```

Comment:
Unfortunately `p2j` isn't clever enough to render these three lines as one paragraph (it inserts line breaks), so I think they need to be all on one line.

Reply:
done